content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -180,19 +181,19 @@ Client: BYE
 Closing connection
 ```
 
-### Client side Programming
-# Establish a Socket Connection
-This is an instance of a socket class and can be obtained in two ways: 
+### Client-side Programming
+#### Establish a Socket Connection
+A Socket instance can be obtained in two ways: 
     
-1. On the server side it can be used as a return value of the accept ( ) method.
+1. On the server side it can be used as a return value of the accept method.
```

Comment:
Change:

>1. On the server side it can be used as a return value of the accept method.
>2. The builder of socket objects accepts the hostname of the server on which the server operates and the server's port number.

To:

>1. Server-side, you would receive it as a return value of the accept method.
>2. You can instantiate it as a Socket object like so:

Grammatical and readability fixes